OpenRA.Mods.Common/Traits/Infantry/TakeCover.cs

```diff
@@ -38,22 +56,25 @@ public class TakeCover : Turreted, INotifyDamage, IDamageModifier, ISpeedModifie
 		public bool IsModifyingSequence { get { return IsProne; } }
 		public string SequencePrefix { get { return info.ProneSequencePrefix; } }
 
+		readonly HashSet<string> proneTriggers;
+
 		public TakeCover(ActorInitializer init, TakeCoverInfo info)
 			: base(init, info)
 		{
 			this.info = info;
+			proneTriggers = info.ProneTriggers.ToHashSet();
```

Comment:
Well how much this actually helps depends on the number of elements inside, so it's pretty iffy on that point, but definitely won't hurt, and IMO would be nice to set a standard.